packages/notifi-dapp-example/tailwind.config.ts

```diff
@@ -17,14 +17,35 @@ const config: Config = {
       rota: ['var(--font-rota)'],
     },
     extend: {
+      width: {
+        86: '343px',
+        112: '445px',
+        18: '70px',
+        90: '360px',
+      },
+      height: {
+        18: '70px',
+      },
+      boxShadow: {
+        container: '0px 10px 40px 0px #0000000D',
+        card: '0px 2px 6px 0px #0000001A',
+      },
       colors: {
         notifi: {
           // TODO: Temp bg color. TBD
           'page-bg': 'rgb(var(--notifi-page-bg))',
           'button-primary-bg': 'rgb(var(--notifi-button-primary-bg))',
+          'button-primary-blueish-bg':
+            'rgb(var(--notifi-button-primary-blueish-bg))',
           'button-primary-text': 'rgb(var(--notifi-button-primary-text))',
           'label-connect-wallet-text':
             'rgb(var(--notifi-label-connect-wallet-text))',
+          'container-bg': 'rgb(var( --notifi-bgimg-left))',
+          'card-bg': 'rgb(var(--notifi-card-bg))',
+          dusk: 'rgb(128, 130, 157)',
```

Comment:
Just my idea. The `notifi-` color config is in components basis so that it will consume the `global css color variable`. This way, we will be able to switch mode easily. 
furthermore, It might be confusing for potential dev if we put color variable in both globall.css & tailwind.config.
Shall we move this to `global.css`?